src/App.css

```diff
@@ -62,3 +61,44 @@
 .underline {
     text-decoration: underline;
 }
+
+.line-height-adjustment {
+    text-align: left;
+    margin-inline-start: 0.6em;
+    margin-block-start: 2em;
+
+    #lineHeightInputLeft {
```

Comment:
Have you intentionally duplicated the styling between `#lineHeightInputLeft` and `#lineHeightInputRight`?

Perhaps they could just both have the same class 🤔